### DISCUSSION TITLE
Add checkbox to search page for "Include Archive content"

### DISCUSSION THREAD

**Discussion by MEGAMENE** [Radiokot/photoprism-android-client#93] [Ideas]
It would be nice to have a checkbox like the "Include private content" checkbox for Archive content in the search page.
Maybe also have an option to just search for the Archive content too.

**Comment by Radiokot** [comment]
Can you do it in the web app? I tried experimenting with search filters, and it seems you can only use `archived:true/false` which returns either achieved or not achieved entries but never both.
